controllers/swagger.go

```diff
@@ -49,6 +49,10 @@ func (s *SwaggerController) serveSwaggerJSON(c *fireball.Context) (fireball.Resp
 				Name:        "Image",
 				Description: "Methods for Images",
 			},
+			{
```

Comment:
This should have been in master - I just did a bad merge yesterday so adding this here.